specs/corla.pvs

```diff
@@ -1053,23 +1052,32 @@ BEGIN
 
 %%% \subsubsection{Selection of Audited Contests}
 
-%%% NO LATER THAN 5:00 P.M. MT ON THE FRIDAY AFTER ELECTION DAY, THE
-%%% SECRETARY OF STATE WILL SELECT FOR AUDIT AT LEAST ONE STATEWIDE
-%%% CONTEST, AND FOR EACH COUNTY AT LEAST ONE COUNTYWIDE CONTEST. THE
-%%% SECRETARY OF STATE WILL SELECT OTHER BALLOT CONTESTS FOR AUDIT IF
-%%% IN ANY PARTICULAR ELECTION THERE IS NO STATEWIDE CONTEST OR A
-%%% COUNTYWIDE CONTEST IN ANY COUNTY. THE SECRETARY OF STATE WILL
-%%% PUBLISH A COMPLETE LIST OF ALL AUDITED CONTESTS ON THE AUDIT
-%%% CENTER. THE SECRETARY OF STATE WILL CONSIDER THE FOLLOWING FACTORS
-%%% IN DETERMINING WHICH CONTESTS TO AUDIT:
-%%% (1) THE CLOSENESS OF THE REPORTED TABULATION OUTCOME OF THE CONTESTS;
-%%% (2) THE GEOGRAPHICAL SCOPE OF THE CONTESTS;
-%%% (3) ANY CAUSE FOR CONCERN REGARDING THE ACCURACY OF THE REPORTED
-%%% TABULATION OUTCOME OF THE CONTESTS;
-%%% (4) ANY BENEFITS THAT MAY RESULT FROM OPPORTUNISTICALLY AUDITING
-%%% CERTAIN CONTESTS; AND
-%%% (5) THE ABILITY OF THE COUNTY CLERKS TO COMPLETE THE AUDIT BEFORE
-%%% THE CANVASS DEADLINE.
+%%% No later than 5:00 PM MT on the Friday after Election Day, the
+%%% Secretary of State will select for audit at least one statewide
+%%% contest, and for each county at least one countywide contest. The
+%%% Secretary of State will select other ballot contets for audit if
```

Comment:
I think this was the only section in all caps... is it the only section that is still a draft? My inclination is that readability should trump fidelity to the source material here, especially since we're not changing the wording.